### PR TITLE
PE-77: Add NVM to development environment with instructions to use it

### PR DESCRIPTION
**Pull request by fedellen**
This PR simply adds an `.nvmrc` file with instructions on installing/using NVM to the README.

Note: The README is currently missing some steps for development setup and has several inaccuracies that are not within the scope of this ticket. Those issues are meant to be addressed in the upcoming PE-78 task. This ticket's scope is limited to setting up and using NVM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ To use the ArDrive CLI, install it with your favorite package manager, and run "
 
 ## Setup
 
+### NVM
+
+This repo uses NVM and an `.nvmrc` file to lock the Node version to the current version used by `ardrive-core-js`. Follow these steps to get NVM up and running on your system:
+
+1. Install NVM using [these installation instructions][nvm-install].
+2. Navigate to this project's root directory (`ardrive-cli`)
```
Comment (arielmelendez) [inline README.md:16]: Specifying that `ardrive-cli` is the root folder name might be presumptuous; users can choose any root folder name for a git repo checkout.

**Review by arielmelendez — APPROVED** [review]
Left a minor comment, but otherwise LGTM

**Comment by fedellen** [comment]
> Left a minor comment, but otherwise LGTM

Good point, its better than I remove that. Updated on last commit

**Review by arielmelendez — APPROVED** [review]
👍

**Review by agsuy — APPROVED** [review]

**Pull request merged**